transformers/component_tests/queries/test_helpers/test_helpers.go

```diff
@@ -184,9 +184,9 @@ func IlkSnapshotFromValues(ilk, updated, created string, ilkValues map[string]in
 		Spot:          ilkValues[vat.IlkSpot].(string),
 		Line:          ilkValues[vat.IlkLine].(string),
 		Dust:          ilkValues[vat.IlkDust].(string),
-		Chop:          ilkValues[cat.IlkChop].(string),
-		Lump:          ilkValues[cat.IlkLump].(string),
-		Flip:          ilkValues[cat.IlkFlip].(string),
+		Chop:          ilkValues[v1_0_10.IlkChop].(string),
+		Lump:          ilkValues[v1_0_10.IlkLump].(string),
+		Flip:          ilkValues[v1_0_10.IlkFlip].(string),
```

Comment:
nbd but may want to use `v1_0_0` in all these places and/or extract shared code to a higher level namespace